requirements.txt

```diff
@@ -2,10 +2,12 @@ azureml-core>=1.15.0
 azureml-dataprep
 azureml-mlflow
 azureml-widgets
+azureml-contrib-services
```

Comment:
move this to the notebook -  e.g.:

```python
!pip install --upgrade azureml-contrib-services tensorflow
```

I'm also concerned that this is using packages in contrib, what for?

Reply:
It uses it for inferencing(azureml.contrib.services.aml_request / aml_response).
So, I see some ambiguity in your instructions, I thought you were asking to move everything that is needed to requirements.txt, and now back to the notebook. Is there some kind of a rule that clarifies it?

Reply:
Moving everything to the notebook.